build.cake

```diff
@@ -166,11 +166,17 @@ Task ("NuGet-Push")
 	.IsDependentOn("Nuget")
 	.Does (() =>
 {
+	var iOSFullVersion = IOSVERSION;
+	
+	if (IOS_SERVICERELEASE_VERSION != "0") {
+		iOSFullVersion = $"{IOSVERSION}.{IOS_SERVICERELEASE_VERSION}";
+	}
+
 	// Get the path to the packages
-	var modelPackage = "./nuget/pkgs/PSPDFKit.iOS.Model." + IOSVERSION +".nupkg";
-	var uiPackage = "./nuget/pkgs/PSPDFKit.iOS.UI." + IOSVERSION +".nupkg";
-	var instantPackage = "./nuget/pkgs/PSPDFKit.iOS.Instant." + IOSVERSION +".nupkg";
-	var macModelPackage = "./nuget/pkgs/PSPDFKit.Mac.Model." + MACVERSION +".nupkg";
+	var modelPackage = $"./nuget/pkgs/PSPDFKit.iOS.Model.{iOSFullVersion}.nupkg";
+	var uiPackage = $"./nuget/pkgs/PSPDFKit.iOS.UI.{iOSFullVersion}.nupkg";
+	var instantPackage = $"./nuget/pkgs/PSPDFKit.iOS.Instant.{iOSFullVersion}.nupkg";
+	var macModelPackage = $"./nuget/pkgs/PSPDFKit.Mac.Model.{MACVERSION}.nupkg";
```

Comment:
Also I do not think this is needed at all since NuGet will always be Name.1.2.3.pkg, the fourth digit is only used by NuGet metadata itself

Reply:
That's weird since I tested this quite a bit today and if I include the `IOS_SERVICERELEASE_VERSION`, even if it is 0, it will be included in the filename, so it searched for e.g. PSPDFKit-9.3.2.0.pkg instead of PSPDFKit-9.3.2.pkg, that's the whole point of why I did this check. For some reason the nuget pack command ignores this in it's `version` property. I don't know why though.

Reply:
So what I'm saying is that the package name will have a fourth digit e.g. 9.3.0.2, so we do need this check for service release versions.

Reply:
Ok with me then :)